torch/distributed/fsdp/_state_dict_utils.py

```diff
@@ -57,23 +57,25 @@ def _convert_to_wrapped_module_name(module_name: str) -> str:
     return module_name
 
 
-def _param_fqns(
+def _param_name_infos(
     module: nn.Module, fsdp_state: _FSDPState
 ) -> Iterator[Tuple[str, str, str]]:
     if not _has_fsdp_params(fsdp_state, module):
         return
     for param_name, module_name in _module_handles(fsdp_state, module)[
         0
-    ].parameter_module_names():
+    ].param_module_names():
         module_name = _convert_to_wrapped_module_name(module_name)
         fqn = f"{module_name}{param_name}"
         yield fqn, param_name, module_name
 
 
-def _shared_param_fqns(module: nn.Module, fsdp_state) -> Iterator[Tuple[str, str, str]]:
+def _shared_param_name_infos(
+    module: nn.Module, fsdp_state
+) -> Iterator[Tuple[str, str, str]]:
```

Comment:
Is it easier to make things like Tuple[str, str, str] into a dataclass so users know exactly what each item means?